src/main/java/com/github/hcsp/reflection/MapBeanConverter.java

```diff
@@ -12,7 +13,33 @@
     //  2. 通过反射获得它包含的所有名为getXXX/isXXX，且无参数的方法（即getter方法）
     //  3. 通过反射调用这些方法并将获得的值存储到Map中返回
     public static Map<String, Object> beanToMap(Object bean) {
-        return null;
+        Class beanClass = bean.getClass();
+        Map<String, Object> map = new HashMap<>();
+        Object value = null;
+        for (Method method : beanClass.getDeclaredMethods()) {
+            if (method.getName().startsWith("get")) {
+                helpBeantoMap(method, "get", value, bean, 3, map);
+            }
+            if (method.getName().startsWith("is")) {
+                helpBeantoMap(method, "is", value, bean, 2, map);
+            }
+        }
+        return map;
+    }
+
+    public static void helpBeantoMap(Method method,
+                                     String startWord,
+                                     Object value,
+                                     Object bean,
+                                     int subStringIndex, Map map) {
+        if (method.getName().startsWith(startWord)) {
+            try {
+                value = method.invoke(bean);
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+            map.put(method.getName().substring(subStringIndex), value);
```

Comment:
```suggestion
			String name = method.getName().substring(subStringIndex);
			String nameWithLowerCase = Character.toLowerCase(name.charAt(0)) + name.substring(1);
            map.put(nameWithLowerCase, value);
```

这里需要把首字母小写。例如`getAbc()`这个getter方法对应的是`abc`属性。